rpm/dokku.postinst

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set +o posix # rpm uses /bin/sh that forces bash in posix mode
+set -eo pipefail; [[ $TRACE ]] && set -x
+
+readonly DOKKU_ROOT="${DOKKU_ROOT:-/home/dokku}"
+readonly DOKKU_LIB_ROOT="${DOKKU_LIB_PATH:-/var/lib/dokku}"
+
+call-sshcommand() {
+  if [[ -x /usr/local/bin/sshcommand ]]; then
+    /usr/local/bin/sshcommand "$@"
+  elif [[ -x /usr/bin/sshcommand ]]; then
+    /usr/bin/sshcommand "$@"
+  else
+    echo "Unable to find sshcommand binary" 1>&2
+    exit 1
+  fi
+}
+
+echo "Setting up dokku user"
```

Comment:
Any reason why this section isn't all in a function at least? The debian one isn't because of the diff, though this all should be.